### PR TITLE
fix: add files before git commit

### PR DESCRIPTION
**Pull request by WilcoFiers**

### CODE REVIEW

File: .github/workflows/data-update.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: Axe data update
 on:
+  workflow_dispatch:
```
Comment (WilcoFiers) [inline .github/workflows/data-update.yml:3]: For testing purposes

File: .github/workflows/data-update.yml [hunk 2]
```diff
@@ -20,4 +21,5 @@ jobs:
         with:
           github-token: ${{ secrets.GH_ACCESS_TOKEN }}
           push-branch: main
+          files: reports/.
```
Comment (straker) [inline .github/workflows/data-update.yml:24]: Hu, would have assumed it did this by default. The docs were not clear on that behavior.

**Review by straker — APPROVED** [review]

**Comment by straker** [comment]
Reviewed for security

File: .github/workflows/data-update.yml (hunk 2 above)
Comment (WilcoFiers) [reply .github/workflows/data-update.yml:24]: Yeah, that was my guess too. It did not.

**Pull request merged**